packages/filesystem/src/browser/style/file-dialog.css

```diff
@@ -69,12 +72,24 @@
 
 .dialogContent .theia-NavigationBack:focus,
 .dialogContent .theia-NavigationForward:focus,
-.dialogContent .theia-NavigationHome:focus {
+.dialogContent .theia-NavigationHome:focus,
+.dialogContent .theia-NavigationUp:focus,
+.dialogContent .theia-LocationInputToggle:focus
+{
     outline: none;
     border: none;
     box-shadow: none;
 }
 
+.dialogContent .theia-NavigationBack.theia-mod-disabled:focus,
+.dialogContent .theia-NavigationForward.theia-mod-disabled:focus,
+.dialogContent .theia-NavigationHome.theia-mod-disabled:focus,
+.dialogContent .theia-NavigationUp.theia-mod-disabled:focus,
+.dialogContent .theia-LocationInputToggle.theia-mod-disabled:focus
+{
+    opacity: var(--theia-mod-disabled-opacity) !important;
```

Comment:
Why does this need the `!important` annotation? I think it's preferable to avoid that, if it can be avoided.

Reply:
Unfortunately this rule is overriding one that is already specified as !important. In this case the icons' opacity is being overridden to retain their disabled appearance even when clicked/focused on